arabic/dream/dream.md

```diff
@@ -1,3 +1,5 @@
 هل تريد ان تحلم؟
 احلم......الاحلام تتحقق اذا صدقة النية
 ولكن الاحلام لا تتحقق الا بالتوكل على الله والاخذ بالاسباب
+الان انت فى بداية الطريق ماذا ستفعل
+1- [ستبدأ فى التعلم] (../arabic/learn/learn.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.